integrations/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -90,6 +90,22 @@ public void backquotedRefName(String branchName) throws NessieNotFoundException
     assertThat(sql("DROP BRANCH `%s` IN nessie", branchName)).hasSize(1).containsExactly(row("OK"));
   }
 
+  @ParameterizedTest
+  @ValueSource(
+      strings = {
+        "USE REFERENCE %s IN nessie",
+        "/* leading */ CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
```

Comment:
A few more interesting test cases (but the first new one will fail):

```suggestion
        "/* leading */ CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
        "/* leading /* leading */ */ CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
        "/* leading -- leading \n */ CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
        "-- leading \n CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
        " -- leading \n -- leading \n -- leading \n CREATE BRANCH IF NOT EXISTS %s_other IN nessie",
```

Reply:
added